docs/ctf-bots.md

```diff
@@ -0,0 +1,35 @@
+# CTF Q-bots
```

Comment:
Spatie's [ab-bots](https://github.com/spatiebot/ab-bot) also work well in CTF. It might be worth mentioning it for those who will choose bots for their server or for development.

Reply:
Added info about spatiebot